src/common.h

```diff
@@ -100,14 +100,18 @@ typedef Bytes48 KZGProof;
 
 /** Stores the setup and parameters needed for computing KZG proofs. */
 typedef struct {
-    /** The length of `roots_of_unity`, a power of 2. */
-    uint64_t max_width;
-    /** Powers of the primitive root of unity determined by `SCALE2_ROOT_OF_UNITY` in bit-reversal
-     * permutation order, length `max_width`. */
-    fr_t *roots_of_unity;
-    /** The expanded roots of unity. */
+    /** The size of our multiplicative subgroup (the roots of unity). This is the size of
+     *  the extended domain (after the RS encoding has been applied), so the size of
+     *  the subgroup is FIELD_ELEMENTS_PER_EXT_BLOB. */
+    uint64_t domain_size;
+    /** Roots of unity in bit-reversal permutation order.
+     *  The array contains `domain_size` elements. */
+    fr_t *brp_roots_of_unity;
```

Comment:
For `g1_values_lagrange_brp` few lines below, we have the `brp` as a suffix. Maybe make it consistent with that?

Reply:
I felt like the other roots of unity structures had the adjective as a prefix, so I did the same. Perhaps we can introduce this convention in a separate PR?

Reply:
Sure 👍